src/chainstate/stacks/db/headers.rs

```diff
@@ -324,6 +307,15 @@ impl StacksChainState {
         Ok(ancestors)
     }
 
+    /// Get an ancestor's height given the tip index hash and the ancestor index hash
+    pub fn get_block_height_of_ancestor(
```

Comment:
I think the callers of this method should be able to just call `conn.get_ancestor_block_height`, right? That way, we wouldn't need to add this static function.